contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -928,6 +929,177 @@ checksum(PG_FUNCTION_ARGS)
         PG_RETURN_INT32(result);
 }
 
+Datum
+object_id(PG_FUNCTION_ARGS)
```

Comment:
Please add a Short description on top for inputs and also that it returns NULL for error scenarios